openapi.yaml

```diff
@@ -3274,7 +3843,67 @@ components:
           $ref: '#/components/schemas/generic-identifier'
         name:
           $ref: '#/components/schemas/generic-identifier'
+
+    temporary-rating:
+      type: object
+      additionalProperties: false
+      description: |
+        Data structure for a temporary rating against a segment.
+        Includes a unique ID, start and (optional) end time, and a reason.  
+
+        May include a set of values, which apply if this temporary rating
+        is either an AAR Exception or temporary seasonal rating.  
+
+      properties:
+        _links:
+          allOf:
+          - $ref: '#/components/schemas/hal-links-block'
+          - type: object
+            properties:
+              segment: 
+                $ref: '#/components/schemas/hal-link'
+        id:
+          $ref: '#/components/schemas/generic-identifier'  
+        segment-id: 
+          $ref: '#/components/schemas/generic-identifier'       
+        start-time: 
+          $ref: '#/components/schemas/period-start'
+        end-time: 
+          $ref: '#/components/schemas/period-start'       
+        updated-time:
+          $ref: '#/components/schemas/timestamp'
+        values:
+          $ref: '#/components/schemas/limit-value-set'
+        temporary-aar-exception:
```

Comment:
Why not implement these as separate endpoints? We need to be able to validate them differently. We need interface limits.

Also, this allows for a representation where temporary-aar-exception = true and temporary-seasonal-rating = true.

Reply:
Maybe this a semantic difference in understanding on my part, but I don't actually see them as being different.  I also see this:
temporary-aar-exception = true and temporary-seasonal-rating = true

as being valid.

Reply:
They are implemented very differently in the clearinghouse. temp AAR exception *must* be proposed by LEP if it doesn't have a telemetered forecast, since it should supersede temp table use. 
![image](https://github.com/ge-grid-software/TROLIE/assets/57670484/2641ad7c-9268-49c8-8ff1-1385f36274d6)

Also, there is some uncertainty around the proper validation for AAR exceptions versus temporary seasonal ratings. In particular, we might want to allow the temporary seasonal to go lower than the reasonability bounds defined on the equipment.

Finally, only equipment with a proper ratings obligation can have a temp AAR exception. Any modeled equipment can have seasonal.